modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/ProjectTemplates.java

```diff
@@ -107,7 +107,31 @@
 									FileUtil.getManifestProperty(
 										path.toFile(), "Bundle-Description");
 
-								templates.put(templateName, bundleDescription);
+								if (bundleDescription != null) {
+									URI uri = path.toUri();
+
+									URL url = uri.toURL();
+
+									URL[] urls = {url};
+
+									try (URLClassLoader classLoader =
```

Comment:
Lets not use URLClassloader becaue that will search the parent classloader for resources of which it might find in other places.  Lets just simply use a Jar/Zip Entry class to ask for that specific path for a JarEntry and make sure that it exists.